readthedocs/doc_builder/python_environments.py

```diff
@@ -295,7 +291,7 @@ def install_core_requirements(self):
         ]
 
         if self.config.doctype == 'mkdocs':
-            requirements.append('mkdocs==0.17.3')
+            requirements.append('mkdocs<1.1')
```

Comment:
I haven't seen anything breaking from their changelog since 1.0 :)

https://mkdocs.readthedocs.io/en/latest/about/release-notes/

1.1 is in current development